src/service/repo/signedMessageRepo.ts

```diff
@@ -37,8 +37,8 @@ interface SignedMessageRepoTypes {
     attachments: any,
     replyMessageId?: string
   ) => Promise<any>;
-  getAllSignedChannels: () => Promise<ChannelData[]>;
-  getAllSignedPostNotifications: () => Promise<SignedPostNotification[]>;
+  getAllSignedChannels: (timeStamp: string) => Promise<ChannelData[]>;
+  getAllSignedPostNotifications: (timeStamp: string) => Promise<SignedPostNotification[]>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `timeStamp` parameter to the `getAllSignedChannels` and `getAllSignedPostNotifications` functions is a good improvement. It allows for more efficient data fetching by retrieving only new data since the last fetch, reducing unnecessary network usage and improving performance.



<!-- This is an auto-generated comment by OSS CodeRabbit -->